tests/integration/testsuites/v2alpha1/testsuite.go

```diff
@@ -128,16 +128,12 @@ func (t *testsuite) Setup() error {
 		return err
 	}
 
-	time.Sleep(time.Duration(t.config.ReqDelay) * time.Second)
```

Comment:
This was there since the beginning of those tests, so maybe it is not required any longer.
This sleep I'd say is a safeguard to make sure deletion is complete. Fun fact is, that the test namespace name is random and we usually never have to delete it. So I think we are good to remove this.
The second sleep is a left-over that was not removed during Hydra removal.
So I think all is good.

Reply:
thanks for clarifying